contrib/cirrus/setup.sh

```diff
@@ -25,5 +25,5 @@ set -x  # show what's happening
 curl --fail --location -o /tmp/netavark.zip "$NETAVARK_URL"
 mkdir -p /usr/libexec/podman
 cd /usr/libexec/podman
-unzip /tmp/netavark.zip
-chmod a+x /usr/libexec/podman/netavark
+#unzip -q /tmp/netavark.zip
+#chmod a+x /usr/libexec/podman/netavark
```

Comment:
For aarch64, unzipping netavark.zip errored out on waiting for `replace netavark`. So, I commented this out.

Reply:
Right, so the problem with this is there is no arm64 build to restore, only x86, so that ain't gonna work.  Let me take a peek and refresh my memory how this all works.